web/src/components/Feedback.tsx

```diff
@@ -87,28 +104,69 @@ const Feedback = ({
   return (
     <Container fullWidth={isSearchFeedback}>
       {isSearchFeedback ? (
-        <InputSection id='searchTerm' title={t('searchTermDescription')}>
-          <Input id='searchTerm' value={searchTerm} onChange={setSearchTerm} />
-        </InputSection>
+        <FormControl error={submitted && !searchTerm} variant='outlined'>
+          <FormHelperText id='searchTerm'>
+            {submitted && !searchTerm ? t('noteFillFeedback') : t('searchTermDescription')}
+          </FormHelperText>
+          <InputLabel htmlFor='searchTerm' />
+          <Input
+            id='searchTerm'
+            value={searchTerm}
+            aria-describedby={searchTerm}
+            onChange={e => setSearchTerm(e.target.value)}
+            fullWidth
+            required
+          />
+        </FormControl>
       ) : (
-        setRating && <FeedbackButtons rating={rating} setRating={setRating} />
+        <FormControl error={submitted && rating === null}>
+          <FeedbackButtons rating={rating} setRating={setRating} />
+          {submitted && rating === null && <FormHelperText>{t('noteFillFeedback')}</FormHelperText>}
+        </FormControl>
       )}
-
-      <InputSection
-        id='comment'
-        title={t(commentTitle)}
-        description={t('commentDescription', { appName: buildConfig().appName })}
-        showOptional>
-        <Input id='comment' value={comment} onChange={onCommentChanged} rows={DEFAULT_ROWS_NUMBER} />
-      </InputSection>
-
-      <InputSection id='email' title={t('contactMailAddress')} showOptional>
-        <Input id='email' value={contactMail} onChange={onContactMailChanged} />
-      </InputSection>
-      <PrivacyCheckbox language={language} checked={privacyPolicyAccepted} setChecked={setPrivacyPolicyAccepted} />
-      {submitFeedbackDisabled && <Note text={t(feedbackFilled ? 'noteFillFeedback' : 'notePrivacyPolicy')} />}
+      <FormControl fullWidth variant='outlined'>
+        <InputLabel htmlFor={comment}>{t(commentTitle)}</InputLabel>
```

Comment:
<img width="839" height="245" alt="Image" src="https://github.com/user-attachments/assets/ef301b5a-eb2c-4266-a0fc-553760eefd8a" />

I think this should be same as the input ID